minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -121,15 +123,21 @@ public synchronized void start() throws Exception {
     mState = State.STARTED;
 
     mMasterAddresses = generateMasterAddresses(mNumMasters);
-    if (zkEnabled()) {
-      mCuratorServer = mCloser
-          .register(new TestingServer(-1, AlluxioTestDirectory.createTemporaryDirectory("zk")));
-      mProperties.put(PropertyKey.ZOOKEEPER_ADDRESS, mCuratorServer.getConnectString());
-    } else {
-      MasterNetAddress masterAddress = mMasterAddresses.get(0);
-      mProperties.put(PropertyKey.MASTER_HOSTNAME, masterAddress.getHostname());
-      mProperties.put(PropertyKey.MASTER_RPC_PORT, Integer.toString(masterAddress.getRpcPort()));
-      mProperties.put(PropertyKey.MASTER_WEB_PORT, Integer.toString(masterAddress.getWebPort()));
+    switch (mDeployMode) {
+      case ZOOKEEPER_HA:
+        mCuratorServer = mCloser
+            .register(new TestingServer(-1, AlluxioTestDirectory.createTemporaryDirectory("zk")));
+        mProperties.put(PropertyKey.ZOOKEEPER_ENABLED, "true");
+        mProperties.put(PropertyKey.ZOOKEEPER_ADDRESS, mCuratorServer.getConnectString());
+        break;
+      case NON_HA:
```

Comment:
Could you put these in alphabetical order?

Reply:
done